corporate/lib/stripe.py

```diff
@@ -1399,6 +1400,9 @@ def process_initial_upgrade(
                 # to worry about this plan being used for any other purpose.
                 # NOTE: This is the 2nd plan for the customer.
                 plan_params["status"] = CustomerPlan.NEVER_STARTED
+                plan_params[
+                    "invoicing_status"
+                ] = CustomerPlan.INVOICING_STATUS_INITIAL_INVOICE_TO_BE_SENT
```

Comment:
Do we need a migration to update existing plans with `status=CustomerPlan.NEVER_STARTED` to have this value?

Reply:
Yeah, right!
Added a migration file to update `invoicing_status` and `next_invoice_date` for legacy plans scheduled to upgrade.